Scripts/Runtime/Core/CollectionsRegistry.cs

```diff
@@ -121,11 +122,18 @@ public List<ScriptableObjectCollection> GetCollectionsByItemType(Type targetColl
             return result;
         }
 
+
+        [Obsolete("Use GetCollectionByGUID(ULongGuid guid) is obsolete, please regenerate your static class")]
```

Comment:
Duplication of the warning text. Just a nitpick

Reply:
Since this should be removed soon, I will not bother wit it now 😆